src/app/reporting/reporting.h

```diff
@@ -41,6 +41,33 @@
 
 #include <app/util/af-types.h>
 
+namespace chip {
+namespace app {
+
+enum class ListOperation
+{
+    NotList,
```

Comment:
When would NotList be used?

Reply:
Not used in this PR, but used in pending https://github.com/project-chip/connectedhomeip/pull/11705. These two definitions need to be merged later.